File: 백준/Bronze/5063. TGN/TGN.cc
#include <stdio.h>

int main(){
    int n, r, e, c;
    scanf("%d", &n);
    for(int i=0; i < n; i++){
        scanf("%d %d %d", &r, &e, &c);
        if(e - (r + c) > 0){printf("advertise\n");}
        else if(e - (r + c) < 0){printf("do not advertise\n");}
        else if(e - (r + c) == 0){printf("does not matter\n");}
    }

    return 0;
}

File: 백준/Bronze/5063. TGN/TGN.java
import java.util.Scanner;

public class Main{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int r,e,c;
        for(int i=0; i < n; i++){
        r = sc.nextInt(); e = sc.nextInt(); c = sc.nextInt();
        if(e - (r + c) > 0){
            System.out.printf("advertise\n");
            
        }
        else if(e - (r + c) < 0){
            System.out.printf("do not advertise\n");
            
        }
        else if(e - (r + c) == 0){
            System.out.printf("does not matter\n");
            
        }
        }
    }
}